scripts/lib/CIME/case/check_lockedfiles.py

```diff
@@ -92,7 +92,7 @@ def check_lockedfile(self, filebase):
                                          "case.build --clean-all and rebuilding")
                     else:
                         self.set_value("BUILD_STATUS", 1)
-                    f2obj.set_value("BUILD_COMPLETE", False)
+
```

Comment:
This is an example of an unflushed write error. We are still vulnerable to this kind of problem.